extensions/interactions/ItemSelectionInput/directives/oppia-interactive-item-selection-input.component.ts

```diff
@@ -157,10 +165,23 @@ export class InteractiveItemSelectionInputComponent implements OnInit {
     this.newQuestion = false;
     this.selectionCount = Object.keys(this.userSelections).filter(
       (obj) => this.userSelections[obj]).length;
-    this.preventAdditionalSelections = (
-      this.selectionCount >= this.maxAllowableSelectionCount);
-    this.notEnoughSelections = (
-      this.selectionCount < this.minAllowableSelectionCount);
+    if (this.minAllowableSelectionCount === this.maxAllowableSelectionCount) {
+      if (this.selectionCount < this.maxAllowableSelectionCount) {
+        this.exactSelections = true;
+        this.preventAdditionalSelections = (
+          this.selectionCount >= this.maxAllowableSelectionCount);
+      } else {
+        this.exactSelections = false;
+        this.preventAdditionalSelections = true;
+      }
+    } else if (this.minAllowableSelectionCount !==
```

Comment:
Does this need an else if here? Just an else should suffice.
Do you also need to set `exactSelections` to false here?

Reply:
Done